awserr.go

```diff
@@ -33,5 +33,12 @@ func IsAWSErrExtended(err error, code string, message string, origErrMessage str
 	if !IsAWSErr(err, code, message) {
```

Comment:
I'm noticing that the comment up top (line 27) seems sort of misleading — the function is returning false if there's no AWS error, an error message if it's not nil, and an empty string if there's an error with no message.